solutions/functions/string-repeat.js

```diff
@@ -0,0 +1,37 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+    idx++;
+    return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+// Do not change the function signature
+// Fill in the function 
+// string_to_repeat will be string like 'abc'
+// repeat will be an int like 3
+// return 'abc abc abc'
+
+function stringRepeat(inputString, repeat){
+    let resultString = "";
+
+    for(let i = 1; i < repeat; i++){
+        resultString += inputString + " ";
+    }
+
+    resultString += inputString;
```

Comment:
@Abduliitr Please add a comment that for the last repetition, we want to avoid the trailing space.   Some typos have been fixed in the template code. So, please take the latest template (you may have to open in incognito to prevent getting cached page).

Reply:
Added the comment and updated as per latest template.